src/kpatch_patch.c

```diff
@@ -480,11 +484,11 @@ kpatch_apply_patches(kpatch_process_t *proc)
 	 * TODO(pboldin): close the holes so the state is the same
 	 * after unpatch
 	 */
-	ret = object_unapply_patch(o, /* check_flag */ 1);
-	if (ret < 0) {
+	if (object_unapply_patch(o, /* check_flag */ 1) < 0) {
```

Comment:
Please, don't hesitate to use more lines. There is no need to short it like this, let's keep the code a single statement a line and let compiler do its job.

Reply:
This way value of 'ret' is not changed if object_unapply_patch was successful, and error code from object_apply_patch is returned. If object_unapply_patch returns error, ERROR_PATCH_UNPATCH will be returned to notify user.